qdrant-landing/content/articles/vector-search-filtering.md

```diff
@@ -0,0 +1,701 @@
+  }
+}
+```
+The `id` is a unique identifier for the point in your collection. The `vector` is a mathematical representation of similarity to other points in the collection. 
+Finally, the `payload` holds metadata that directly describes the point.
+
+Though we may not be able to decipher the vector, we are able to derive additional information about the item from its metadata, In this specific case, **we are looking at a data point for a laptop that costs $899.99**. 
+
+## What is filtering?
+
+When searching for the perfect computer, your customers may end up with results that are mathematically similar to the search entry, but not exact. For example, if they are searching for **laptops under $1000**, a simple [vector search](/advanced-search/) without constraints might still show laptops over $1000. 
+
+This is why [semantic search](/advanced-search/) alone **may not be enough**. In order to get the exact result, you would need to enforce a payload filter on the `price`. Only then can you be sure that the search results abide by the chosen characteristic.
+
+> This is called **filtering** and it is one of the key features of [vector databases](https://qdrant.tech). 
+
+Here is how a **filtered vector search** looks behind the scenes. We'll cover its mechanics in the following section.
+
+![vector-search-filtering](/articles_data/vector-search-filtering/vector-search-filtering.png)
+
+#### Want to see the result? Keep reading!
+The filtered result will be a combination of the semantic search and the filtering conditions imposed upon the query. In the following pages, we will show that **filtering is a key practice in vector search for two reasons:** 
+
+1. With filtering, you can **dramatically increase search precision**. More on this in the next section.</br>
+2. Filtering helps control resources and **reduce compute use**. More on this in [**Payload Indexing**](#filtering-with-the-payload-index).
+
+## What you will learn in this guide:
+
+In [vector search](/advanced-search/), filtering and sorting are more interdependent than they are in traditional databases. While databases like SQL use commands such as `WHERE` and `ORDER BY`, the interplay between these processes in vector search is a bit more complex.
+
+Most people use default settings and build vector search apps that aren't properly configured or even setup for precise retrieval. In this guide, we will show you how to **use filtering to get the most out of vector search** with some basic and advanced strategies that are easy to implement. 
+
+#### Remember to run all tutorial code in Qdrant's Dashboard 
+
+The easiest way to reach that "Hello World" moment is to [**try filtering in a live cluster**](/documentation/quickstart-cloud/). Our interactive tutorial will show you how to create a cluster, add data and try some filtering clauses. 
+
+![qdrant-filtering-tutorial](/articles_data/vector-search-filtering/qdrant-filtering-tutorial.png)
+
+## Qdrant's approach to filtering 
```

Comment:
Maybe we could mention somewhere that we are working with dense vectors here.

Sparse vector indexing and filtering works differently :)